### DISCUSSION TITLE
Resolved: Turning lanes suggested an intersection too early (Warszawa Ursynów, Pileckiego)

### DISCUSSION THREAD

**Discussion by etam** [osmandapp/OsmAnd#15416] [Maps Related]
On this route: https://osmand.net/map?start=52.146417%2C21.036624&end=52.142763%2C21.039215&mode=car#17/52.14480/21.03953 OsmAnd suggests turning lanes too early.

![Screenshot_20221003-162726](https://user-images.githubusercontent.com/220752/193607126-d3a44200-6ea0-4a7f-8f6f-2633d7e9de80.png)

I guess it's not the problem with OsmAnd itself, but rather error in OpenStreetMap data. I tried looking into it myself, but I lack knowledge and experience on how to do it.

**Comment by tommycrock** [comment]
The turn lanes didn't carry on across the intersection to the point where the left turn was so osmand was treating straight on as left. Hopefully fixed here: https://www.openstreetmap.org/changeset/126952093

**Comment by etam** [comment]
Yes, it's fixed. Thanks!

![Screenshot_20221004-084547](https://user-images.githubusercontent.com/220752/193752734-fed01f39-4428-450c-8f82-d4970b864907.png)
